nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/new-parameter-context-dialog.jsp

```diff
@@ -31,13 +31,15 @@
                     <div class="setting">
                         <div class="setting-name">Name</div>
                         <div id="parameter-context-name-container" class="setting-field">
-                            <input type="text" id="parameter-context-name" name="parameter-context-name"/>
+                            <input type="text" id="parameter-context-name" class="edit-mode" name="parameter-context-name"/>
+                            <div id="parameter-context-name-read-only" class="read-only"></div>
```

Comment:
This needs ellipsis and a title attr for the tooltip. The Id field will also need this...

![Screen Shot 2019-09-10 at 9 21 44 AM](https://user-images.githubusercontent.com/6797571/64617916-46138c80-d3ad-11e9-807f-c200443766ce.png)